vimdoc/module.py

```diff
@@ -60,11 +60,17 @@ def Merge(self, block, namespace=None):
       # Overwrite existing section if it's a default.
       if block_id not in self.sections or self.sections[block_id].IsDefault():
```

Comment:
Yeah, it's a matter of traversal order. Special flags like `plugin[foo]` can have a default from one file (plugin/foo.vim) and explicit value declared in another, so we don't know which definition will come last and try to "overwrite" the other.